sktime/forecasting/compose/_multiplexer.py

```diff
@@ -147,24 +150,49 @@ def fit(self, y, X=None, fh=None, **fit_params):
         self : returns an instance of self.
         """
 
-        self._is_fitted = False
-
-        self._set_y_X(y, X)
-        self._set_fh(fh)
         self._check_forecasters()
         self._set_forecaster()
         forecaster_fit_params = self._check_fit_params(fit_params=fit_params)
         self._forecaster.fit(y, X=X, fh=fh, **forecaster_fit_params)
-        self._is_fitted = True
         return self
 
     def _predict(self, fh, X=None, return_pred_int=False, alpha=DEFAULT_ALPHA):
+        """Forecast time series at future horizon.
+
+        Parameters
+        ----------
+        fh : int, list, np.array or ForecastingHorizon
+            Forecasting horizon
+        X : pd.DataFrame, optional (default=None)
+            Exogenous time series
+        return_pred_int : bool, optional (default=False)
+            If True, returns prediction intervals for given alpha values.
+        alpha : float or list, optional (default=DEFAULT_ALPHA)
+
+        Returns
+        -------
+        y_pred : pd.Series
+            Point predictions
+        y_pred_int : pd.DataFrame - only if return_pred_int=True
+            Prediction intervals
+        """
         return self._forecaster.predict(
             fh, X, return_pred_int=return_pred_int, alpha=alpha
         )
 
-    def update(self, y, X=None, update_params=True):
-        """Call predict on the forecaster with the best found parameters. """
+    def _update(self, y, X=None, update_params=True):
+        """Call predict on the forecaster with the best found parameters.
+
+        Parameters
+        ----------
+        y : pd.Series
+        X : pd.DataFrame, optional (default=None)
+        update_params : bool, optional (default=True)
+
+        Returns
+        -------
+        self : an instance of self
+        """
         self.check_is_fitted()
         self._update_y_X(y, X)
```

Comment:
This can go, right?
```suggestion
```

Reply:
indeed, but as always, also please check in `BaseForecaster` whether there's corresponding code there.

Reply:
Yes it is